amp/WEB-INF/src/org/digijava/module/aim/action/CompareActivityVersions.java

```diff
@@ -1,12 +1,12 @@
 package org.digijava.module.aim.action;
 
-import java.lang.reflect.Field;
+//import java.lang.reflect.Field;
```

Comment:
for code tidiness instead of commenting out unused imports we should just remove them